collect_app/src/main/java/org/odk/collect/android/utilities/CustomTabHelper.java

```diff
@@ -93,8 +95,10 @@ public void openUri(Context context, Uri uri) {
             customTabsIntent.intent.setPackage(getPackageName(context).get(0));
             customTabsIntent.launchUrl(context, uri);
         } else {
-            //open in an external browser
-            context.startActivity(new Intent(Intent.ACTION_VIEW, uri));
+            //open in webview
+            Intent intent = new Intent(context, WebViewActivity.class);
+            intent.putExtra("url",uri.toString());
```

Comment:
There is no checkstyle for that? I mean it should be a space before `uri.toString() `. Please fix.

Reply:
There is no checkstyle for that, I tried running it but it didn't show any such issue.
But will make the changes.